pkg/handler-launcher-com/cmd/v1/cmd.proto

```diff
@@ -115,6 +119,7 @@ message VirtualMachineOptions {
   ClusterConfig clusterConfig = 7;
   map<string, string> interfaceDomainAttachment = 8;
   map<string, InterfaceBindingMigration> interfaceMigration = 9;
+  map<string, FeaturesToDisable> cpuModelToFeaturesToDisable = 10;
```

Comment:
I do wonder why we need `map<string, FeaturesToDisable>`, isn't `FeaturesToDisable` enough?

Reply:
Do you mean just setting a CPU Feature to disable with every CPU Model?